README.md

```diff
@@ -1,3 +1,68 @@
 # Cloud SDK
 
 SDK to be used to access Pasqal Cloud Services.
+
+## Installation
+
+To install the latest release of the `cloud-sdk`, have Python 3.8.0 or higher installed, then use pip:
```

Comment:
Would be worth testing this assumption. We use pretty basic features, I'm sure we also support lower versions of python. 
Not in the scope of this PR though

Reply:
We actually don't support python 3.7 because of the typings module. For example, `Literal` is new in python 3.8 (see (here)[https://docs.python.org/3/library/typing.html#typing.Literal] ) and we use it for our endpoints. I initially tried it with python 3.7 and I couldn't use the SDK.

It looks like on pulser they added this to their `setup.py` : 

```
    extras_require={
        ":python_version == '3.7'": [
            "backports.cached-property",
            "typing-extensions",
        ],
    },
```

I am wondering if this fixes the issue

Reply:
ok, nvm